doc/compute-job-configuration.md

```diff
@@ -0,0 +1,390 @@
+
+Each _Report Configuration_ includes the following data:
+- Topology
+- Metric Profile
+- Availability Profile
+- Weights _(optional)_
+- Downtimes _(optional)_
+
+<!--
+Each tenant has its own folder which contains job subfolders. Each job subfolder contains daily supplementary data files from argo-connectors. The directory structure resembles the following:
+
+- path_to_argo_connector_data
+  + tenantA
+    - Job_Critical
+    - Job_All
+-->
+[](Comment: I have removed this part because it describes the input data)
+
+## Hadoop client configuration
+
+In order for the engine to be able to connect and submit jobs successfully in a hadoop cluster, proper hadoop client configuration files must be present the installed node (`/etc/hadoop/conf/`)
+
+## ARGO Compute Engine configuration files
+
+The main configuration file of the ARGO Compute Engine component is installed by default at `/etc/ar-compute-engine.conf`. In addition, a directory with supplementary secondary configuration files is created in `/etc/ar-compute/`
+
+### /etc/ar-compute-engine.conf
+
+The main configuration files includes various global parameters used by the engine which are organized in sections, as described next:
+
+#### `[default]`
+
+- `mongo_host` (Required)
+
+	Specify the ip address of the datastore node (running mongodb)
+- `mongo_port`(Required)
+
+	Specify the port number of the datastore node (running mongodb)
+- `mode` (Required)
+
+   The mode the engine runs. There are two available options: _cluster_ and _local_:
+
+	 - If the mode is specified as _cluster_, the engine runs connecting to an existing hadoop cluster. It expects that that the hadoop client is properly installed and configured.
+	 - If the mode is specified as _local_, the engine runs local node.
+
+- `serialization` (Required)
+
+	The serialization type used. There are two available options _avro_ and _none_:
+
+	- If specified as _avro_, the engine expects metric and sync data in avro format.
+	- If specified as _none_, the engine expects to find metric and sync data in simple text file delimited format.
+
+- `prefilter_clean`(Required])
+
+	Boolean. If the local prefilter file will be automatically cleaned after hdfs upload or not. If set to _true_ local prefilter file will be automatically cleaned after hdfs upload
+
+- `sync_clean`(Required)
+
+   Bolean. If the uploaded sync files will be automatically cleaned after a job completion or not. If set to _true_ uploaded sync files will be automatically cleaned after a job completion
+
+[](Comment: Need to better explain what prefilter_clean and sync_clean are)
```

Comment:
I suggest that we make these two flags (prefilter_clean and sync_clean) to be false by default in the CE internals and have them as optional configuration flags. @kaggis ?